### ISSUE TITLE
Feature Request: Get Left/Right desktop into pipeline without switching

### ISSUE THREAD

**Issue by kard02** [MScholtes/VirtualDesktop#21]
Is it possible with the current implementation? Maybe I'm missing something.

**Comment by MScholtes** [comment]
Use something like (example for left desktop)
```
VirtualDesktop.exe GetCurrentDesktop
set /a LEFTDESKTOP=%ERRORLEVEL%-1
echo %LEFTDESKTOP%
```

In the environment variable LEFTDESKTOP you get the number of the left desktop you can use with following VirtualDesktop.exe calls.

Greetings

Markus

**Comment by kard02** [comment]
Thanks for the quick response!
I will use that approach, but I still think it would be useful to create `/LeftNoSwitch` or something like that. But decision is up to you, close this issue if you don't want to implement that.

**Comment by MScholtes** [comment]
Hello kard02,

I released the new version 1.8 of **_VirtualDesktop_** today that has a new parameter /CALC to add or substract from the number in the pipeline. For example you can switch to "last" desktop with
```
Virtualdesktop.exe Count Calc:-1 Switch
```
new.

Hope that is what you need.

Greetings

Markus

**Comment by kard02** [comment]
Thanks for the feature!
